### PR TITLE
🍰 2019/kw22/alpha_data_import_status_schema_split

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

- fixed several errors handling import
- split graphql schema into parts
- which data is imported which is not - a list

### Issues

- relates https://github.com/Human-Connection/Human-Connection/issues/692

### Todo

- [x] catalogue all the data not yet started to be imported
- [ ] import the data (will not be done in this PR)
- [ ] refactor graphql schema (will not be done in this PR)

### CODE REVIEW

File: backend/src/types/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  var results = []
+  var list = fs.readdirSync(dir)
+  list.forEach(file => {
+    file = path.join(dir, file).toString('utf-8')
+    var stat = fs.statSync(file)
+    if (stat && stat.isDirectory()) {
+      // Recurse into a subdirectory
+      results = results.concat(findGqlFiles(file))
+    } else {
+      if (file.split('.').pop() === 'gql') {
```
Comment (roschaefer) [inline backend/src/types/index.js]: I guess there is a cleaner way to check for a file extension.

**Comment by ulfgebhardt** [comment]
@roschaefer the given error makes no sense
Regarding to the neo4j [docu](https://neo4j.com/developer/kb/how-do-i-authenticate-with-cypher-shell-without-specifying-the-username-and-password-on-the-command-line/) the neo4j username and password are automatically set from environment variables called `NEO4J_USERNAME` and `NEO4J_PASSWORD` - the command uses those and the `.env` file does not override it by default.

Therefore they are either missing in your call or the database does not require authentication in that container?!

File: backend/src/types/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        // Is a gql file
+        results.push(file)
+      }
+    }
+  })
+  return results
+}
```
[the remark this answers is not in the document]
Comment (ulfgebhardt) [reply backend/src/types/index.js]: yes

**Comment by roschaefer** [comment]
@ulfgebhardt I don't know why, but I added unrelated commits by accident, let me rebase that..

**Pull request merged**